apps/frontend/src/plugins/mitt/index.ts

```diff
@@ -111,3 +111,6 @@ export function mitt<Events extends Record<EventType, unknown>>(
         }
     };
 }
+
+const emitter = mitt();
```

Comment:
The `emitter` is then instanciated twice. Here and in `react-mitt`.
Must keep it as it was.